cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -209,13 +216,14 @@ private void copyStream(ReadOnlyDataSource from, DataSource to, String fromName,
     private static final String FORMAT = "CGMES";
 
     public static final String ALLOW_UNSUPPORTED_TAP_CHANGERS = "iidm.import.cgmes.allow-unsupported-tap-changers";
+    public static final String BOUNDARY_LOCATION = "iidm.import.cgmes.boundary-location";
     public static final String CHANGE_SIGN_FOR_SHUNT_REACTIVE_POWER_FLOW_INITIAL_STATE = "iidm.import.cgmes.change-sign-for-shunt-reactive-power-flow-initial-state";
     public static final String CONVERT_BOUNDARY = "iidm.import.cgmes.convert-boundary";
     public static final String CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE = "iidm.import.cgmes.create-busbar-section-for-every-connectivity-node";
+    public static final String POST_PROCESSORS = "iidm.import.cgmes.post-processors";
     public static final String POWSYBL_TRIPLESTORE = "iidm.import.cgmes.powsybl-triplestore";
+    public static final String PROFILE_USED_FOR_INITIAL_FLOWS_VALUES = "iidm.import.cgmes.profile-used-for-initial-flows-values";
```

Comment:
I prefer `PROFILE_USED_FOR_STATE_HYPOTHESIS` instead of `PROFILE_USED_FOR_INITIAL_FLOWS_VALUES`
I prefer `iidm.import.cgmes.profile-used-for-state-hypothesis` instead of  `iidm.import.cgmes.profile-used-for-initial-flows-values`.

Reply:
Done